editor/property_selector.cpp

```diff
@@ -409,10 +425,12 @@ void PropertySelector::_hide_requested() {
 
 void PropertySelector::_notification(int p_what) {
 	switch (p_what) {
+		case 30: {
```

Comment:
@AThousandShips I don't know what `case 30` corresponds to https://github.com/godotengine/godot/blob/master/scene/main/node.h#L364-L413 (its not listed)? it seems to emit from canceling the PropertySelector window closing it with the x in the top right and confirming selection with the `Open` button (is it like a default for dialogue), am I allowed to name this 30 signal and if so, what to name it ?

Reply:
`30` is `NOTIFICATION_DRAW` in `CanvasItem`, see:
https://github.com/godotengine/godot/blob/97b8ad1af0f2b4a216f6f1263bef4fbc69e56c7b/scene/main/canvas_item.h#L188

But I don't know why this was added like this

Reply:
Oh okay, I probably shouldn't be using it to clear properties then :thinking: